### PR TITLE
WIP: Add Redirect widget

### PR DESCRIPTION
**Pull request by gregorylegarec**
First draft for intents Redirect widget.

Does not work as CozyClient is undefined in `redirect` method.

### CODE REVIEW

File: src/components/Redirect.jsx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+      this.setState({
+        parsingError: false
+      })
+      return parsed
+    } catch (e) {
+      this.setState({
+        parsingError: true
+      })
+      return toParse
+    }
```
Comment (ptbrowne) [inline src/components/Redirect.jsx]: This method is called in the constructor. I am not sure that doing setState in the constructor is supported.

**Comment by nono** [comment]
Ping @gregorylegarec

**Comment by nono** [comment]
Ping @gregorylegarec

**Comment by gregorylegarec** [comment]
I will not terminate this WIP before I leave, but it could be used as a basis for someone else to implement the feature.

I suggest to close it and keep a track of it into [this issue](https://github.com/cozy/cozy-sandbox/issues/166)

**Pull request closed without merging**